src/routes/api/trips.routes.js

```diff
@@ -136,5 +136,5 @@ router.post('/request', [checkUser, dateValidator, tripRequestValidator], tripsC
  *         description: Travel request successfully created
  */
 router.post('/multi-city', validateMultiCity, checkUser, tripsController.createMultiCityTrip);
-
+router.get('/request', checkUser, tripsController.getTrips);
```

Comment:
@blaise82  I've created middleware called ```isProfileUpdated.js```. You must accept it While you'll be rebasing.